packages/smithy-core/src/smithy_core/interfaces/identity.py

```diff
@@ -1,35 +1,22 @@
 #  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 #  SPDX-License-Identifier: Apache-2.0
-from datetime import datetime
-from typing import Protocol, TypedDict, TypeVar, runtime_checkable
+from datetime import UTC, datetime
+from typing import Protocol, runtime_checkable
 
 
 @runtime_checkable
 class Identity(Protocol):
     """An entity available to the client representing who the user is."""
 
-    # The expiration time of the identity. If time zone is provided,
-    # it is updated to UTC. The value must always be in UTC.
     expiration: datetime | None = None
+    """The expiration time of the identity.
+
+    If time zone is provided, it is updated to UTC. The value must always be in UTC.
```

Comment:
We say this in a bunch of places but I'm not sure I've seen an implementation where we actually do this. Is this a note to implementers, or should we have some base implementation that checks timezones and makes adjustments?

Reply:
The identity resolver is expected to do that. I could update this to a dataclass and drop it in post_init?